tests/functional/codegen/calling_convention/test_default_parameters.py

```diff
@@ -191,17 +193,17 @@ def bar(a: int128, b: int128 = -1) -> (int128, int128):  # noqa: E501
 
     c = get_contract(code)
 
-    assert c.bar(-123) == [-123, -1]
-    assert c.bar(100, 100) == [100, 100]
+    assert c.bar(-123) == (-123, -1)
+    assert c.bar(100, 100) == (100, 100)
 
     def validate_value(cls, value):
         pass
 
     monkeypatch.setattr("eth_abi.encoding.NumberEncoder.validate_value", validate_value)
 
-    assert c.bar(200, 2**127 - 1) == [200, 2**127 - 1]
-    with tx_failed():
-        c.bar(200, 2**127)
+    assert c.bar(200, 2**127 - 1) == (200, 2**127 - 1)
+    with tx_failed(EncodeError):
+        c.bar(200, 2**127, disambiguate_signature="bar(int128,int128)")
```

Comment:
why does this need disambiguate_signature? i don't think it was failing with encoder error before, was it?

Reply:
Without disambiguate we raise `Exception: Could not find matching bar function for given arguments.`
I missed the patch a couple lines before. I will adapt this to ignore the unsigned validation and fail with Revert.